code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -366,7 +366,7 @@ var/global/list/parasites = list() //all currently existing/living guardians
 	var/list/possible_guardians = list("Chaos", "Standard", "Ranged", "Support", "Explosive", "Lightning", "Protector", "Charger", "Assassin")
 	var/random = TRUE
 	var/allowmultiple = 0
-	var/allowling = 0
+	var/allowling = 1
```

Comment:
Because it makes good sense? Because its annoying having to balance every ling power and traitor item around this 1% grief lotto jackpot winner.